parts/k8s/cloud-init/artifacts/cse_config.sh

```diff
@@ -678,10 +678,4 @@ cleanUpContainerd() {
 removeEtcd() {
   rm -rf /usr/bin/etcd
 }
-removeMoby() {
```

Comment:
Moving these funcs to cse_install because that's the only shell script file that VHD CI sources, so we always have these (in fact innocuous) errors during VHD builds:

`/home/packer/provision_installs.sh: line 93: removeMoby: command not found`

`/home/packer/provision_installs.sh: line 90: removeContainerd: command not found`